docs/src/pages/components/stack/IntroStack.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Stack from '@material-ui/core/Stack';
+import Box from '@material-ui/core/Box';
+
+function Cell({ children }: { children: React.ReactNode }) {
+  return <Paper sx={{ p: 2, color: 'text.secondary' }}>{children}</Paper>;
+}
+
+export default function IntroStack() {
```

Comment:
For naming consistency with the other components?

```suggestion
export default function BasicStack() {
```

(and rename the file too)?